addons/lunch/models/lunch_order.py

```diff
@@ -1,7 +1,8 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, models, _
+
+from odoo.osv.expression import AND
 from odoo.exceptions import ValidationError, UserError
```

Comment:
really nitpicking shoud come after exceptions and kept groupted

here is the best practices
In Python, the recommended order for importing modules is split in three groups:

    Standard Library Imports: These should come first, though none are in your case.
    Third-Party Imports: External libraries or dependencies.
    Local Application/Library Specific Imports: Specific to your application.

within a group if two import depends on the same lib the best way is by level of deepness and then by alphabetic order

```py
from odoo import api, fields, models, _
from odoo.exceptions import ValidationError, UserError
from odoo.osv.expression import AND
```